<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="soccf-tr5-001" name="occasional-tests" tests="9" failures="1" timestamp="2 Jan 2015 09:13:28 GMT" time="183.659" errors="0">
  <testcase name="testStableSort" time="0.765" classname="teammates.test.cases.ui.browsertests.TableSortTest"/>
  <testcase name="testTableSortingDate" time="0.204" classname="teammates.test.cases.ui.browsertests.TableSortTest"/>
  <testcase name="testTableSortingDiff" time="0.577" classname="teammates.test.cases.ui.browsertests.TableSortTest"/>
  <testcase name="testTableSortingID" time="0.218" classname="teammates.test.cases.ui.browsertests.TableSortTest"/>
  <testcase name="testTableSortingName" time="1.295" classname="teammates.test.cases.ui.browsertests.TableSortTest"/>
  <testcase name="testTableSortingPoint" time="0.218" classname="teammates.test.cases.ui.browsertests.TableSortTest"/>
  <testcase name="testAll" time="5.507" classname="teammates.test.cases.ui.browsertests.SystemErrorEmailReportTest"/>
  <testcase name="executeJsTests" time="8.83" classname="teammates.test.cases.ui.browsertests.AllJsTests">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
	at teammates.test.cases.ui.browsertests.AllJsTests.executeJsTests(AllJsTests.java:47)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- executeJsTests -->
  <testcase name="loadWebpageCompilation" time="91.979" classname="teammates.test.cases.ui.browsertests.MashupPageUiTest"/>
</testsuite> <!-- occasional-tests -->
